compliance_checker/cf/util.py

```diff
@@ -327,15 +328,28 @@ def download_cf_standard_name_table(version, location=None):
     if location is None:  # This case occurs when updating the packaged version from command line
         location = resource_filename('compliance_checker', 'data/cf-standard-name-table.xml')
 
-    url = "http://cfconventions.org/Data/cf-standard-names/{0}/src/cf-standard-name-table.xml".format(version)
-    r = requests.get(url, allow_redirects=True)
-    if r.status_code == 200:
-        print("Downloading cf-standard-names table version {0} from: {1}".format(version, url), file=sys.stderr)
-        with open(location, 'wb') as f:
-            f.write(r.content)
+    if version == 'latest':
+        from bs4 import BeautifulSoup
```

Comment:
I've always been confused by in-line imports like this. Not saying this is bad, but I've seen sources in the past say that inline imports are organizationally bad. It seems that it actually makes more sense to have them like this if the option is rarely used.

Reply:
The consensus from the Python community seems to be that it's OK to use it's for a feature that's not used all the time.  For example, Pandas has certain code paths for Excel output of a dataframe which appear to import the libraries when the exporting function is called.  

In this particular case, since people won't necessarily always be updating their standard name tables every time, it's not necessary to import the library for BeautifulSoup every time.

Reply:
What's pep8 have to say about it?

Reply:
Hmm, my linter didn't consider that an error.  I'm going to rewrite this code to use plain lxml without the BeautifulSoup extension.